src/core/sys/freebsd/netinet/in_.d

```diff
@@ -0,0 +1,581 @@
+//Written in the D programming language
```

Comment:
I think that Walter likes people to put in on their files for searchability when google comes trauling along...